Essentials/src/main/resources/config.yml

```diff
@@ -341,24 +341,30 @@ pastebin-createkit: false
 # Color is not an actual sign, it's for enabling using color codes on signs, when the correct permissions are given.
 
 enabledSigns:
-  #- color
+  #- anvil
```

Comment:
These entries should probably be sorted either grouped by functionality, *or* alphabetical, not a mixture of both